src/goto-programs/goto_clean_expr.cpp

```diff
@@ -260,13 +260,15 @@ void goto_convertt::clean_expr(
       // preserve the expressions for possible later checks
       if(if_expr.true_case().is_not_nil())
       {
-        code_expressiont code_expression(if_expr.true_case());
+        code_expressiont code_expression(
```

Comment:
I guess this is C-style `(void)x` to mark something "used"? If so perhaps say that in the comment.